app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+
+  def update
+    current_user.update(params.require(:user).permit!)
```

Comment:
It's not immediately necessary, but you will want to create a proper `user_params` method and permit specific attributes. There is special syntax for the nested attribute stuff, so we can work on that together if you want.